### ISSUE TITLE
Want to print entire namelist to log

### ISSUE THREAD

**Issue by mnlevy1981** [marbl-ecosys/MARBL#16]
Currently, the GCM reads four MARBL namelists (`ecosys_nml`, `ecosys_ciso_nml`, `ecosys_parms_nml`, and `ecosys_restore_nml`) from a file and passes them as a string to MARBL. MARBL sets default values, then updates those values based on what the GCM passes.

Desired behavior: MARBL then writes the entire namelist to status log
Current behavior: MARBL writes the values set in the namelist file to status log

**Comment by mnlevy1981** [comment]
I have been playing around in

https://github.com/mnlevy1981/nml_bugs

And am starting to see some useful information. For starters, I think I need to write the namelists into an array of strings rather than one specific string... which will actually make logging the namelist much easier. For N variables in the namelist, the string needs to be of length at least N+2:

```
nmlstr(1) = "&NAMELIST_NAME"
nmlstr(2) = "VAR_1 = VAL_1,"
nmlstr(3) = "VAR_2 = VAL_2,"
...
nmlstr(N+1) = "VAR_N = VAL_N,"
nmlstr(N+2) = "/"
```

**Comment by mnlevy1981** [comment]
I just created a branch to add a stand-alone driver to MARBL to allow us to test this out. The build for the driver is not very flexible (yet), and requires gfortran to compile, but even if the stand-alone driver is not adopted it'll provide a quick and easy way to test fixes for this issue.

https://github.com/mnlevy1981/MARBL/tree/enhancement/write_full_namelist

**Comment by mnlevy1981** [comment]
I deleted the `write_full_namelist` branch in favor of

https://github.com/mnlevy1981/MARBL/tree/enhancement/initialization

I still think we want to use the stand-alone driver to test out this work, but it sounds like we're going to go the way of #51 and #62 for the fix (rather than writing the entire namelist, marbl will have some sort of `write_parameter_list` function)

**Comment by mnlevy1981** [comment]
This has been implemented in a branch and will be fixed when #80 is merged onto master.
